packages/dds/tree/src/simple-tree/api/viewSchemaToSimpleSchema.ts

```diff
@@ -45,6 +47,7 @@ import { LeafNodeSchema } from "../leafNodeSchema.js";
 export function toSimpleTreeSchema(
 	schema: ImplicitFieldSchema,
 	copySchemaObjects: boolean,
```

Comment:
Now that we have multiple boolean options here, might be a good time to add an "options" property bag that contains them (instead of separate params).